src/storage.ts

```diff
@@ -191,7 +191,7 @@ export interface WorkspaceStorage {
 			dismissed?: Record<string, boolean>;
 		};
 		columns?: Record<string, StoredGraphColumn>;
-		hiddenRefs?: Record<string, StoredGraphHiddenRef>;
+		excludeRefs?: Record<string, StoredGraphExcludedRef>;
```

Comment:
Because this is stored data, I think we should avoid the naming change here. And we don't have a system for good data migration (as vscode doesn't make that easy)

Reply:
Done in commit: [Revert workspace naming change](https://github.com/gitkraken/vscode-gitlens/pull/2388/commits/11c1ea003481d8e67aa1af5c43554a25f34e1d00)